core/src/main/java/org/apache/calcite/plan/RelOptUtil.java

```diff
@@ -2611,12 +2611,12 @@ private static RexShuttle pushShuttle(final Project project) {
    * {@link org.apache.calcite.rel.rules.MultiJoin}.
```

Comment:
The previous line mentions LogicalProject which is no longer true.

Reply:
done